src/njs_parser.c

```diff
@@ -4606,16 +4605,37 @@ njs_parser_statement(njs_parser_t *parser, njs_lexer_token_t *token,
         break;
     }
 
-    ret = njs_parser_statement_wo_node(parser, token, current);
-    if (ret != NJS_OK) {
-        return ret;
-    }
+    njs_parser_next(parser, njs_parser_statement_wo_node);
 
     return njs_parser_after(parser, current, parser->node, 1,
                             njs_parser_statement_after);
 }
 
 
+static njs_int_t
+njs_parser_empty_block_statement(njs_parser_t *parser, njs_lexer_token_t *token,
+    njs_queue_link_t *current)
+{
+    njs_parser_node_t  *node;
+
+    node = njs_parser_node_new(parser, NJS_TOKEN_BLOCK);
+    if (node == NULL) {
+        return NJS_ERROR;
+    }
+
+    node->token_line = parser->line;
+
+    node->left = NULL;
```

Comment:
njs_parser_node_new() does the zeroing, NULL initialization is not needed.
The same for the lines below.